dizoo/taxi/envs/taxi_env.py

```diff
@@ -0,0 +1,192 @@
+import copy
+import os, sys
+from typing import List, Tuple, Union, Literal, Optional
+
+import imageio
+import gym
+from gym.spaces import Space, Discrete
+from gym.spaces.box import Box
+import numpy as np
+from easydict import EasyDict
+
+from ding.envs.env.base_env import BaseEnv, BaseEnvTimestep
+from ding.torch_utils import to_ndarray
+from ding.utils import ENV_REGISTRY
+
+
+@ENV_REGISTRY.register('taxi')
+class TaxiV3Env(BaseEnv):
+    def __init__(self, cfg: dict) -> None:
+        
+        #^ 该部分为初始化定义，需要有的
+        self._cfg = EasyDict(

```

Comment:
use the implementation like `default_config` in other previous environment in dizoo